src/careamics/models/lvae/noise_models.py

```diff
@@ -34,18 +34,67 @@ def noise_model_factory(
         If the chosen noise model `model_type` is not implemented.
         Currently only `GaussianMixtureNoiseModel` is implemented.
     """
-    if model_config:
-        noise_models = []
-        for nm_config in model_config.noise_models:
-            if nm_config.model_type == "GaussianMixtureNoiseModel":
-                noise_models.append(GaussianMixtureNoiseModel(nm_config))
+    # TODO in case path are provided, load. Should they be in the config ?
+    # TODO probably yes, should be a list of models per channel
+    noise_models = []
+    for nm in model_config.noise_models:
+        if nm.path:
+            if nm.model_type == "GaussianMixtureNoiseModel":
+                noise_models.append(GaussianMixtureNoiseModel(nm))
             else:
-                raise NotImplementedError(
-                    f"Model {nm_config.model_type} is not implemented"
-                )
-        return MultiChannelNoiseModel(noise_models)
-    # TODO should config and paths be mutually exclusive? What you mean, Igor?
-    return None
+                raise NotImplementedError(f"Model {nm.model_type} is not implemented")
+
+        else:  # TODO this means signal/obs are provided. Controlled in pydantic model
+            # TODO train a new model. Config should always be provided?
+            if nm.model_type == "GaussianMixtureNoiseModel":
+                # TODO one model for each channel all make this choise inside the model?
+                return train_gm_noise_model(nm)
```

Comment:
Given the current `MultiChannelNoiseModel`, and assuming that this function returns a `GaussianMixtureNoiseModel`, I think here we'd need something like 
```
noise_models.append(train_gm_noise_model)
```